tests/conftest.py

```diff
@@ -91,6 +93,7 @@ def config():
                 "submodules": [
                     {
                         "machine_name": "predators",
+                        "name": "Predatord",
```

Comment:
"Predators"

Reply:
Nice catch.

Reply:
> "Predators"

You can also add suggestions if you like and GitHub will also mark you as a co-author of this PR.

Reply:
I always forget about that feature. Will do in the future. :+1: